src/Tests/Meditation.AttachProcessService.Tests/ProcessSnapshotCreatorTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Meditation.AttachProcessService.Tests
+{
+    public class ProcessSnapshotCreatorTests : TestsBase
+    {
+        [Fact]
+        public async Task ProcessSnapshotCreator_ReturnsSnapshotForDotNetProcess()
+        {
+            // Prepare
+            var processId = Environment.ProcessId;
```

Comment:
[ToFix] Use types instead of ```var``` if the name of type is not specified in right side of expression.